src/timesync/timesyncd-manager.c

```diff
@@ -1131,3 +1138,60 @@ int manager_new(Manager **ret) {
 
         return 0;
 }
+
+static int manager_save_time_handler(sd_event_source *s, uint64_t usec, void *userdata) {
+        Manager *m = userdata;
+
+        assert(m);
+
+        return manager_save_time_and_rearm(m);
```

Comment:
We usually ignore errors on callback functions. Please voidify the return value:
```
(void) manager_save_time_and_rearm(m);
return 0;
```